_data/es/navigation.yml

```diff
@@ -6,7 +6,7 @@ nav:
       subsubfolderitems:
         - page:          Windows
           url:           /wiki/Installation-for-Windows
-        - page:          Macintosh
+        - page:          macOS
           url:           /wiki/Installation-for-Macintosh
         - page:          Linux
           url:           /wiki/Installation-for-Linux
```

Comment:
@ignotus666 The link to the "Onboarding" page seems to be missing for the spanish translation.

Reply:
Done.

Reply:
Great